### ISSUE TITLE
Permanent Attachments

### ISSUE THREAD

**Issue by TR-LawGiver** [GamerGambit/cw2-fas2-weapon-attachment-vendor#4]
Is it possible to make attachments permanent?

**Comment by GamerGambit** [comment]
Yep see here https://github.com/GamerGambit/cw2-fas2-weapon-attachment-vendor?tab=readme-ov-file#database-support

Although this has not been tested in years so let me know how you go.

**Comment by TR-LawGiver** [comment]
Should I use sql to make this happen?

Because although I changed it to "true" it didn't save my attachments.

**Comment by GamerGambit** [comment]
It should use the sqlite database that ships with garrysmod. What game mode are you trying this with? I'll see if I can reproduce it

**Comment by TR-LawGiver** [comment]
Thank you so much for trying to help :) I really appreciate that.

I don't use any sql server and trying to use this vendor in "darkrp" gamemode.

**Comment by GamerGambit** [comment]
Testing locally without a dedicated server I'm not able to reproduce this issue. Did you change `addons/cw2-fas2-weapon-attachment-vendor-master/lua/attachment_vendor/configs/sv_attachment_vendor_config.lua` to look like this?
```lua
ATTACHMENT_VENDOR.persistent = {
   enabled = true,  // Should bought attachments be saved to a database?
   module = "sqlite",// What database module should be used? (CURRENTLY ONLY SQLITE WORKS)
   auth = {          // Authentication for a remote MYSQL server (CURRENTLY NOT USED)
      host = "localhost",
      port = 3306,
      user = "root",
      pass = ""
   }
};```

Also I forgot to ask what weapon pack/s you are using so I can test that those actually work.

**Comment by GamerGambit** [comment]
Glad to hear its working.

Unfortunately I haven't followed mods for Garry's Mod in a long time and am unable to help you with that.
